app/controllers/api/v1/articles_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Api::V1::ArticlesController < ApplicationController
+
+  def show
+    article = Article.find(params[:id])
+    render json: { entries: article }
```

Comment:
I do not think it makes sense to have the article inside of a key called `entries`. 
Can you have `render json: article`

Reply:
yes